pkg/minikube/minikube.go

```diff
@@ -154,6 +155,32 @@ func checkForExistingCluster() error {
 		fmt.Scanf("%s", &resp)
 		if strings.ToLower(resp) != "y" {
 			fmt.Println("Installation skipped")
+			checkKnativeNamespace := exec.Command("kubectl", "get", "namespaces")
+			output, err := checkKnativeNamespace.CombinedOutput()
+			namespaces := string(output)
+			if err != nil {
+				fmt.Println(string(output))
+				return fmt.Errorf("check existing cluster: %w", err)
+			}
+			if strings.Contains(namespaces, "knative") {
+				fmt.Print("Knative installation already exists.\nDelete and recreate the cluster [y/N]: ")
+				fmt.Scanf("%s", &resp)
+				if strings.ToLower(resp) != "y" {
+					fmt.Println("Skipping installation")
+					installKnative = false
+					return nil
+				} else {
+					fmt.Println("deleting cluster...")
+					deleteCluster := exec.Command("minikube", "delete", "--profile", "--insecure-registry", clusterName)
```

Comment:
```suggestion
					deleteCluster := exec.Command("minikube", "delete", "--profile", clusterName)
```

don't need to specify the registry on delete (sorry if I missed this on an earlier review...)